### PR TITLE
Evo 676 reacts on comments

### LINKED ISSUE #676: Change upvoting to emoji reactions
Do we want a limited set like FB or all of them like Slack?

### PR DESCRIPTION
**Pull request by thomasgwatson**
closes #676

### CODE REVIEW

File: src/components/PostListRow/PostListRow.js
Comment (thomasgwatson) [inline src/components/PostListRow/PostListRow.js:47]: Haven't gotten to PostListRow or Grids views. Do we want to try tackling them now as well???

File: src/routes/PostDetail/PostDetail.connector.js
Comment (thomasgwatson) [inline src/routes/PostDetail/PostDetail.connector.js:56]: I don't think any of this is needed but I'm not completely sure how to test. 

There are no props being merged in the 'mergeProps' function, sooo.... delete?

File: src/routes/PostDetail/PostDetail.js
Comment (thomasgwatson) [inline src/routes/PostDetail/PostDetail.js:168]: I think this change is all linting

**Review by tibetsprague — APPROVED** [review]
looks good ! but lets hold off on merging. still waiting on grid items too?

**Comment by lorenjohnson** [comment]
Hey @thomasgwatson -- just catching-up to this. Excited for it to move forward and this all looks good!

Once you we get closer to merging please hmu re. the ORM Reducer merge conflicts and general App considerations. I don't expect they're anything hairy, but I'll be copying at least the reducer changes into the App. 

If we don't find the moment before it's merged that's fine too, but it'd be best to catch it then.

**Comment by tibetsprague** [comment]
already merged with chat

**Pull request closed without merging**
